integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -172,11 +172,15 @@ def client(self):
         if self._client:
             return self._client
 
-        if self._url and self._url.startswith("http") and self._url.endswith(".weaviate.network"):
+        if (
+            self._url
+            and self._url.startswith("http")
+            and (self._url.endswith(".weaviate.network") or self._url.endswith(".weaviate.cloud"))
```

Comment:
When I set up a new cluster in weaviate cloud, I can confirm the url ends with `.weaviate.cloud `